Confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -43,7 +44,7 @@ class Confidence internal constructor(
     private val contextChanges: Flow<Map<String, ConfidenceValue>> = contextMap
         .drop(1)
         .distinctUntilChanged()
-    private val coroutineScope = CoroutineScope(dispatcher)
+    private val coroutineScope = CoroutineScope(SupervisorJob() + dispatcher)
```

Comment:
Without this, network errors or missing client secret would cause this coroutine to crash and prevent many crucial operations relying on it

Reply:
can we move this change out into a separate PR?